app/resources/inventory.py

```diff
@@ -6,6 +6,16 @@
 from vars import game_info
 
 
+@m.user_endpoint(path=["inventory", "trade"], requires=trade_requirements)
+def trade(data: dict, user: str) -> dict:
+    element: Optional[Inventory] = wrapper.session.query(Inventory).filter_by(element_uuid=data["element_uuid"]).first()
```

Comment:
"wrong typing" use Union[Inventory, None]

Reply:
Why? `Optional[Inventory]` is a shortcut for `Union[Inventory, None]` @SpartanerSpaten

Reply:
The thing is Optional is thought for "optional" arguments in functions here you only have 2 possible return types None and Inventory

Reply:
From the official Python documentation:

> You can use `Optional[X]` as a shorthand for `Union[X, None]`.

Reply:
Its personal preference